IFAS/BL/IFBL.py

```diff
@@ -0,0 +1,84 @@
+from DAL import IFDAL
+from BL import IfHistoryBL
+import requests
+import validators
+from tasks.transactionTasks import get_transaction
+import re
+from Config.addones import Status, externalIfAssUrl, ifAssIfUrl, executeIfAssUrl, makeJsonBody
+from logger import logger
+
+
+def getIf(id: str) -> list:
+    newIf = IFDAL.getIf(id)
+    return newIf
+
+def getAllIfs() -> list:
+    allIfs = IFDAL.getAllIfs()
+    return allIfs
+
+def createIf(name: str,properties: str, url: str = None) -> dict:
+    body = {
+        "name": name,
+        "properties":{
+            "expression": properties
+        },
+        "url" : url
+    }
+    requestBody = makeJsonBody(body)
+    if checkIfValidation(name):
+        transactionId = requests.post(externalIfAssUrl, json=requestBody).json()["transactionId"]
```

Comment:
but what will happen if the request.post won't return you transactionId? handle this case